c++/include/orc/Common.hh

```diff
@@ -34,6 +34,7 @@ namespace orc {
   public:
     static const FileVersion& v_0_11();
     static const FileVersion& v_0_12();
+    static const FileVersion& UNSTABLE_PRE_2_0();
```

Comment:
Could you spin off this `UNSTABLE_PRE_2_0`-related part, @stiga-huang ?

Reply:
Sorry that I don't understand this comment. Are you suggesting adding a check for this version instead of introducing an unstable version?

BTW, I use the same name we used in Java: https://github.com/apache/orc/blob/77b4d5acb58e03d50503d1b687421cac627a6e0b/java/core/src/java/org/apache/orc/OrcFile.java#L79

Reply:
I'm suggesting to making another PR for `UNSTABLE_PRE_2.0` handling because it's orthogonal to `Decimal64` read support. If you spin off them, we can backport that PR to branch-1.7, @stiga-huang .

Reply:
Oh, that's a good point! Sure, let me do it.

Reply:
Created the PR: #1064